### DISCUSSION TITLE
Disable excess 'HDR' effect?

### DISCUSSION THREAD

**Discussion by octimis** [smplayer-dev/smplayer#696] [Q&A] [answered]
The images demonstrates two video effects. Top is what SMPlayer looks like and bottom is what MPV standalone looks. What setting do I configure to make SMPlayer look like MPV.

https://i.imgur.com/TPt7Ea5.png

**Comment by smplayer-dev** [comment, marked as answer]
Go to preferences and be sure that mpv is selected as multimedia engine, and select "gpu" as video output.

**Reply by octimis** [reply]
> Go to preferences and be sure that mpv is selected as multimedia engine, and select "gpu" as video output.

In addition to setting multimedia engine to MPV, changing output driver from x11 (Linux) to gpu fixed the problem. Thank you.

**Comment by Hyporeal** [comment]
I had the same issue, but only when paused and definitely if the window moved when paused. MPV was already set. The output driver was set to "default" and I changed it to "gpu" with no effect. It seemed to resolve after changing it to "gpu-next".
